devicetypes/rooms-beautiful/curtain.src/curtain.groovy

```diff
@@ -0,0 +1,260 @@
+/**
+ *
+ *	Copyright 2019 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ */
+
+import groovy.json.JsonOutput
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition(name: "Curtain", namespace: "Rooms Beautiful", author: "Alex Feng", ocfDeviceType: "oic.d.blind", mnmn: "SmartThings", vid: "generic-shade-4") {
+		capability "Actuator"
+		capability "Battery"
+		capability "Configuration"
+		capability "Refresh"
+		capability "Window Shade"
+		capability "Health Check"
+        
+        attribute("replay", "enum")
+        attribute("battLife", "enum")
+
+		command "pause"
```

Comment:
Please go ahead and remove this line `command "pause"` since this is now in the Window Shade capability (but please leave both of the attribute lines above).

Reply:
@dkirker I've now changed this in the DTH and committed it.